docs/security/trust/content_trust.md

```diff
@@ -152,14 +152,36 @@ In an environment where `DOCKER_CONTENT_TRUST` is set, you can use the
 `--disable-content-trust` flag to run individual operations on tagged images
 without content trust on an as-needed basis.
 
-```bash
-$  docker pull --disable-content-trust docker/trusttest:untrusted
+Consider the following Dockerfile that uses an untrusted base image:
+
+```
+$  cat Dockerfile
+FROM docker/trusttest:latest
+RUN echo
+```
+
+In order to build a container successfully using this Dockerfile, one can do:
+
+```
+$  docker build --disable-content-trust -t <username>/nottrusttest:testing .
```

Comment:
Thanks and please don't apologise! I know it's a little confusing and I hope we can make it as straightforward as possible. So it seems that I have made a mistake in the above line i.e 166. It should be `<username>/nottrusttest:latest .` and not `<username>/nottrusttest:testing .` Also, at the end we do need an unsigned image but I am already using the `docker/trusttest:latest` image there since it is unsigned and already available. Anyway what I had in mind was the following:

```
- Build and push an unsigned <username>/nottrusttest:latest image
- Pull unsigned docker/trusttest:latest
- Build and push signed <username>/trusttest:testing image
- Push unsigned username/trusttest image (this is nothing but pushing unsigned <username>/trusttest:latest image)
- Push signed <username>/otherimage:latest
- Pull signed <username>/trusttest:testing (pushed above)
- Pull unsigned docker/trusttest:latest OR pull unsigned <username>/nottrusttest:latest image (pushed above) OR  pull
unsigned <username>/trusttest:latest (pushed above) (since all these are unsigned)
```
And this is what the document is doing right now. I can add (after making it a little more compact maybe?) this as a chart at the beginning but I just wanna know if we are on the same page here before I make any more changes and confuse you even more!